Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1760.cs

```diff
@@ -90,5 +94,18 @@ void CreateHeaderPage()
 				};
 			}
 		}
+
```

Comment:
Based on https://github.com/xamarin/Xamarin.Forms/issues/1760#issuecomment-440766900, we also need a version of this test which uses a StackLayout (not just a ScrollView).